### ISSUE TITLE
Shapespace learning results seems error

### ISSUE THREAD

**Issue by bearprin** [Chumbyte/DiGS#3]
Hi, I have re-trained DiGS under DFAUST using code since there is no pre-trained model.

I ran the official code directly except for some ``path`` changes.
The vis_reslut of epoch 52 are shown below:

<img width="600" alt="image" src="https://user-images.githubusercontent.com/90443362/189521405-45e4d4d4-a2c5-4b20-999b-d329b0038e44.png">

But after epoch 52, the network can not produce vis_resluts until epoch 59, and the vis_reslut of epoch 59 is shown below:

<img width="903" alt="image" src="https://user-images.githubusercontent.com/90443362/189521450-ccb23118-e7b2-49a4-9aae-3a3ac095e5c1.png">

Appentarly, they are some mistakes happened, but I have no idea about it.

At the same time, the latter results are still like above (vis_reslut of epoch  63):

<img width="792" alt="image" src="https://user-images.githubusercontent.com/90443362/189521614-79a5b23b-4a1f-41fe-961d-0dd7599b922d.png">

Could you help me or release the pre-trained model for shape space learning?
Thanks.

**Comment by Chumbyte** [comment]
The link to the pretrained model can be found here: https://github.com/Chumbyte/DiGS/issues/2#issuecomment-1664881399

Hmm yes it seems like until epoch 52 it was properly learning, and then after then it seems to have collapsed. Not sure why it didn't happen when we ran it, but the scaling on the divergence loss seems to be very sensitive for this task. Maybe try increasing it? We didn't have enough to properly investigate it back then, training took around a week and we didn't have enough GPUs.

I'm going to leave this issue open, maybe I can find some time to look into this (and I have better GPU resources now).

Recently, I came across a really interesting paper on arxiv: [StEik: Stabilizing the Optimization of Neural Signed Distance Functions and Finer Shape Representation](https://arxiv.org/abs/2305.18414). It gives an improved version of the divergence loss, and a theoretical analysis of this "stability with regularisation" and otherwise "collapse" issue from the PDE perspective.
